packages/server/graphql/mutations/helpers/createNewMeetingPhases.ts

```diff
@@ -132,6 +133,20 @@ const createNewMeetingPhases = async (
         case LAST_CALL:
         case 'SCOPE':
           return new GenericMeetingPhase(phaseType, durations)
+        case 'RESPONSES':
+          const teamMembers = await dataLoader.get('teamMembersByTeamId').load(teamId)
+          const teamMemberIds = teamMembers.map(({id}) => id)
```

Comment:
If we add a new team member, the meeting will be missing a stage for him. What would be the best place/way to handle that scenario? Same for team member removal.

Reply:
After some investigation I think it'd be good to add this login into: `JoinMeetingMutation` 🤔